### DISCUSSION TITLE
Might Guy Theme - TURZX 5" Inch Smart Screen

### DISCUSSION THREAD

**Discussion by brunoraisson** [mathoudebine/turing-smart-screen-python#303] [Themes]
I tried to leave it as complete as possible, I hope you like it because I did it to my liking. 

Download Link [](url)https://drive.google.com/file/d/1-1Yvs4U09ELpUw44LNvgCvfUEU0yVpTe/view?usp=sharing

![WhatsApp Image 2023-08-08 at 17 43 46](https://github.com/mathoudebine/turing-smart-screen-python/assets/134862982/ff0f7186-ee0d-4280-9052-b9753b6ac989)

https://github.com/mathoudebine/turing-smart-screen-python/assets/134862982/bb2718c2-3942-4999-97c0-1446662c45d2

The fps counter works with rivatuner activated in the image options.

![RivaTuner](https://github.com/mathoudebine/turing-smart-screen-python/assets/134862982/2547681e-11ae-4810-86a3-8805f4f1d80b)

**Comment by DJRoby19** [comment]
Very nice, but I don't understand for which program you did it.
Isn't it for turing-smart-screen-python?

**Reply by brunoraisson** [reply]
thanks, I posted a picture of the display model, take a look, it's the 5 inch turing model.

**Reply by aStroNuts04** [reply]
hi there! im still struggling to get any form of fps showing. how did you get it to work with rivatuner?

**Reply by c0ldJS** [reply]
It's the Turing hardware but you're using a stock panel driver for the theme, not the `turing-smart-screen-python` project like the rest of us.

Unless you have some YAML and a `sensors_custom.py` file to share. You'll also have to share how you got an animated background working under this project.

**Comment by AhmedWasThere** [comment]
Link is down plz update
